README.Rmd

```diff
@@ -0,0 +1,161 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-",
+  out.width = "100%"
+)
+```
+
+# frogger
+
+<!-- badges: start -->
+[![Project Status: Concept – Minimal or no implementation has been done yet, or the repository is only intended to be a limited example, demo, or proof-of-concept.](https://www.repostatus.org/badges/latest/concept.svg)](https://www.repostatus.org/#concept)
+[![R build status](https://github.com/mrc-ide/frogger/workflows/R-CMD-check/badge.svg)](https://github.com/mrc-ide/frogger/actions)
+[![codecov.io](https://codecov.io/github/mrc-ide/frogger/coverage.svg?branch=main)](https://codecov.io/github/mrc-ide/frogger?branch=main)
+<!-- badges: end -->
+
+Leapfrog is a multistate population projection model for demographic and HIV epidemic estimation.
+
+The name _leapfrog_ is in honor of [Professor](https://iussp.org/en/basia-zaba-1949-2018) Basia [Zaba](https://translate.google.co.uk/?sl=pl&tl=en&text=Zaba&op=translate).
```

Comment:
Note the updates URLs here, looks like original ones from leapfrog no longer work :(